src/rtc/core.py

```diff
@@ -152,6 +153,304 @@ def save_as_cog(filename, scratch_dir = '.', logger = None,
 
 
 
+def _get_ogr_polygon(min_x, max_y, max_x, min_y, file_srs):
+    file_ring = ogr.Geometry(ogr.wkbLinearRing)
+    file_ring.AddPoint(min_x, max_y)
+    file_ring.AddPoint(max_x, max_y)
+    file_ring.AddPoint(max_x, min_y)
+    file_ring.AddPoint(min_x, min_y)
+    file_ring.AddPoint(min_x, max_y)
+    file_polygon = ogr.Geometry(ogr.wkbPolygon)
+    file_polygon.AddGeometry(file_ring)
+    file_polygon.AssignSpatialReference(file_srs)
+    assert file_polygon.IsValid()
+    return file_polygon
+
+
+def get_tile_srs_bbox(tile_min_y_utm, tile_max_y_utm,
+                      tile_min_x_utm, tile_max_x_utm,
+                      tile_srs, polygon_srs, logger=None):
+    """Get tile bounding box for a given spatial reference system (SRS)
+
+       Parameters
+       ----------
+       tile_min_y_utm: float
+              Tile minimum Y-coordinate (UTM)
+       tile_max_y_utm: float
+              Tile maximum Y-coordinate (UTM)
+       tile_min_x_utm: float
+              Tile minimum X-coordinate (UTM)
+       tile_max_x_utm: float
+              Tile maximum X-coordinate (UTM)
```

Comment:
Based on burst database, some high-latitude area uses polar stereographic. Therefore I think we should use more general term than `UTM`. How about replacing `UTM` into `map coordinate?`